tests/ha/test_multi_server_pod_failure.py

```diff
@@ -629,3 +629,350 @@ def test_writes_till_kserver_pods_fail(self):
+        LOGGER.info("Step 6: Verified status for In-flight WRITEs while %s (K) server pods "
+                    "going down.", self.kvalue)
+
+        LOGGER.info("Step 7: Create multiple buckets and run IOs")
+        self.test_prefix = 'test-40575-1'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix, nsamples=2,
+                                                    nclients=2)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 7: Successfully created multiple buckets and ran IOs")
+        LOGGER.info("ENDED: Test to verify continuous WRITEs during %s (K) server "
+                    "pods down - unsafe shutdown", self.kvalue)
+
+    # pylint: disable=too-many-locals
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-40574")
+    @CTFailOn(error_handler)
+    def test_conti_reads_during_kserver_pods_down(self):
+        """
+        This test tests continuous READs while pods are failing till K
+        server pods down - unsafe shutdown
+        """
+        LOGGER.info("STARTED: Test to verify continuous READs during %s (K) "
+                    "server pods down - unsafe shutdown", self.kvalue)
+        LOGGER.info("Step 1: Perform WRITEs with variable object sizes.")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-40574'
+        self.s3_clean.update(users)
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix, skipread=True,
+                                                    skipcleanup=True, nclients=5, nsamples=5)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 1: Performed WRITEs with variable sizes objects.")
+
+        LOGGER.info("Step 2: Perform READs/VerifyDI on written data in background")
+        output = Queue()
+        event = threading.Event()
+        args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
+                'nclients': 1, 'nsamples': 5, 'skipwrite': True, 'skipcleanup': True,
+                'output': output}
+
+        thread = threading.Thread(target=self.ha_obj.event_s3_operation,
+                                  args=(event,), kwargs=args)
+        thread.daemon = True  # Daemonize thread
+        thread.start()
+        LOGGER.info("Step 2: Successfully started READs/VerifyDI on written data in background")
+
+        LOGGER.info("Step 3: Shutdown %s (K) server pods one by one during READs/VerifyDI on"
+                    "written data in background", self.kvalue)
+        LOGGER.info("Get server pod names to be deleted")
+        pod_list = self.node_master_list[0].get_all_pods(pod_prefix=const.SERVER_POD_NAME_PREFIX)
+        self.pod_name_list = random.sample(pod_list, self.kvalue)
```

Comment:
Use system_random instead of random at all places.
import secrets
cls.system_random = secrets.SystemRandom()

Reply:
This will taken care in next PR